package.json

```diff
@@ -20,7 +20,9 @@
     "tslib": "^2.2.0",
     "tslint": "^6.1.3",
     "typescript": "^4.3.5",
-    "xmldom": "^0.6.0"
+    "xmldom": "^0.6.0",
+    "builtin-modules": "^3.2.0",
```

Comment:
Thank you for drafting this to illustrate your point!

Would you be willing to say more about what build errors you were seeing without these being present?

Reply:
I am not familiar with JS/TS development, so I might make stupid mistakes here.
```
$ node --version
v17.2.0
$ npm --version
8.1.4
```


Without "builtin-modules", here is my screenshot
![image](https://user-images.githubusercontent.com/7410579/163855450-536adadc-77f3-4708-ad8e-02a74c4cb737.png)


Further without "yarn", here it is
![image](https://user-images.githubusercontent.com/7410579/163855244-b54e0f08-d789-437d-8bbe-6c46b9b3f3fb.png)

Reply:
Interesting, thank you! I could perhaps make it more explicit in the Readme that `yarn` is expected to be installed globally (as in its [main installation instructions](https://classic.yarnpkg.com/lang/en/docs/install/#debian-stable)). I haven't seen the `builtin-modules` error before, but I will keep an eye out for it going forward, as well, and am certainly willing to add it here if it comes up again.

Reply:
You are totally right. The "builtin-modules" issue is because that I am running "npm run de" instead of "yarn dev". Thanks!